index.html

```diff
@@ -544,36 +544,36 @@ <h2 class="icon icon-download">Download CouchDB 3.3.3</h2>
 
         <ul class="list download-list">
           <li>
-            <a href="https://dlcdn.apache.org/couchdb/source/3.3.3/apache-couchdb-3.3.3.tar.gz" class="type">Source</a>
+            <a href="https://dlcdn.apache.org/couchdb/source/3.4.0/apache-couchdb-3.4.0.tar.gz" class="type">Source</a>
             <span class="info">
-              Version 3.3.3 |
-              <a href="https://docs.couchdb.org/en/latest/whatsnew/3.3.html#version-3-3-3" class="release">Release Notes</a> |
-              <a href="https://www.apache.org/dist/couchdb/source/3.3.3/apache-couchdb-3.3.3.tar.gz.asc">PGP Signature</a> |
-              <a href="https://www.apache.org/dist/couchdb/source/3.3.3/apache-couchdb-3.3.3.tar.gz.sha256">SHA256</a> |
+              Version 3.4.0 |
+              <a href="https://docs.couchdb.org/en/latest/whatsnew/3.4.html#version-3-4-0" class="release">Release Notes</a> |
+              <a href="https://www.apache.org/dist/couchdb/source/3.4.0/apache-couchdb-3.4.0.tar.gz.asc">PGP Signature</a> |
+              <a href="https://www.apache.org/dist/couchdb/source/3.4.0/apache-couchdb-3.4.0.tar.gz.sha256">SHA256</a> |
             </span>
           </li>
           <li>
             <a href="https://neighbourhood.ie/download-apache-couchdb-win/" class="type">
               Windows (x64)
             </a>
             <span class="info">
-              Erlang/OTP 24.3.4.17 | Version 3.3.3 |
-              <a href="https://docs.couchdb.org/en/latest/whatsnew/3.3.html#version-3-3-3" class="release">Release Notes</a>
+              Erlang/OTP 25.3.2.13 | Version 3.4.0 |
```

Comment:
Will need to update around the time of the announcement (cc: @big-r81)

Reply:
Yup, that is the included Erlang/OTP version.